src/config.h.cmake.in

```diff
@@ -1,3 +1,6 @@
+#ifndef __GLOG_CONFIG_H__
+#define __GLOG_CONFIG_H__
```

Comment:
Macro names with prefix underscores are reserved for compiler use. Just change the define into `GLOG_CONFIG_H`. That should do just fine.